test/e2e/steps/modifyTask.step.js

```diff
@@ -0,0 +1,47 @@
+const TaskAsserts = require('../asserts/Task.assert');
+const Context = require('../../data/Context');
+const LoginPage = require('../pages/LoginPage');
+const { Given, When, Then } = require('cucumber');
+const TaskPage = require('../pages/TaskPage');
+const DashboardPage = require('../pages/DashboardPage');
+
+Given(/^user is in home page$/, function () {
+  LoginPage.open(Context.page.endpoints.base);
+  DashboardPage.listViewHeader.waitForExist();
+});
+
+When(/^user creates a task$/, function (dataTable) {
+  let task = dataTable.rowsHash();
+  DashboardPage.floatingMenuButton.click();
+  TaskPage.setFloatingName(task.name);
+  TaskPage.setFloatingDescription(task.description);
+  TaskPage.setPriorityButton.click();
+  TaskPage.priorityOption(task.priority.toLowerCase()).click();
+  TaskPage.floatingStatusButton.click();
+  TaskPage.floatingTaskStatus(task.status.toLowerCase()).click();
+  TaskPage.createTaskButton.click();
+});
+
+When(/^user modifies "([^"]*)" fields$/, function (taskName, dataTable) {
+  let task = dataTable.rowsHash();
+  TaskPage.getTaskInList(taskName).waitForExist(3000);
+  TaskPage.getTaskInList(taskName).click();
+  let elm = $('.task-name__overlay').getText().length;
+  TaskPage.modalName.click();
+  for(let i = 0; i < elm; i++) {
+    browser.keys('\uE003');
```

Comment:
Please, use utils class for implementing the keys

Reply:
This ASCII Code was moved to SystemInteractions 714a80d40d2cac7bdd711b382977570e9e1cad4a